src/export-transactions.ts

```diff
@@ -101,8 +108,9 @@ async function getTransactions(name: string, tokenAddress: string, network: Netw
       ) ? TransferType.INTERNAL : txTransfer.transfer_type
 
       const date = tx.block_signed_at.split('T')[0]
-      const usdPrice = priceData[txTransfer.contract_address.toLowerCase()][date]
-      const amount = parseNumber(Number(txTransfer.delta), txTransfer.contract_decimals)
+      const contractAddress = txTransfer.contract_address.toLowerCase()
+      const usdPrice = fullFetch ? priceData[contractAddress][date] : BALANCES[contractAddress]?.rate
+      const amount = parseNumber(Number(txTransfer.delta), BALANCES[contractAddress]?.decimals || txTransfer.contract_decimals)
```

Comment:
If someone tries to run this scripts locally for the first time, will the `balances.json` file exist?
I'm wondering if we should import RAW_BALANCES like `require('../public/balances.json')` and try/catch it in the assignation 
`const BALANCES = (RAW_BALANCES as BalanceParsed[]).reduce`.
Or what happens if the import fails...

Reply:
The scripts are intended to be executed in the same order as assigned in the github actions:

<img width="498" alt="image" src="https://user-images.githubusercontent.com/45410089/226442405-7b3c1450-54e6-4a61-b5aa-6aea6381e743.png">

This can be checked at [pull-data.yml](https://github.com/Decentraland-DAO/transparency/blob/main/.github/workflows/pull-data.yml)

Reply:
Roger dat

![image](https://user-images.githubusercontent.com/2858950/226443384-658203a0-f252-49af-b479-642ef23470c9.png)